concourse/tasks/run-task.sh

```diff
@@ -30,6 +30,11 @@ COMMAND=${COMMAND:-"$(cat "run-task-command/run-task-command")"}
 : "${CLUSTER:?CLUSTER not set}"
 : "${VARIANT:?VARIANT not set}"
 
+if [[ "${SKIP_DB_MIGRATIONS:-false}" == "true" ]]; then
+  echo "Skipping DB Migrations"
```

Comment:
Could we rename this to something more generic, since the task might not be a db migration? This script is also used for running generic tasks in ECS, such as smoke tests.

Perhaps `DISABLE` or `DRY_RUN` would be more appropriate here.

Reply:
aye aye, good idea.